common/djangoapps/student/views.py

```diff
@@ -443,6 +443,7 @@ def signin_user(request):
         # pipeline, if any.
         'pipeline_running': 'true' if pipeline.running(request) else 'false',
         'pipeline_url': auth_pipeline_urls(pipeline.AUTH_ENTRY_LOGIN, redirect_url=redirect_to),
+        'ec_src': request.session.get('ec_src', None),
```

Comment:
We should move this into the enterprise_helpers.py library -- the function could take the request.session object and return the source.  This way we keep the knowledge/awareness of enterprise things from spreading throughout the system, and can re-use the same logic in different places (such as on L475 below)